Assets/StreamingAssets/LUA/Furniture.lua

```diff
@@ -31,8 +31,14 @@ function OnUpdate_GasGenerator( furniture, deltaTime )
         end
     end
 	return
+	furniture.SetAnimationState("running")
 end
 
+function OnPowerOff_GasGenerator( furniture, deltaTime )
```

Comment:
Is the convention to have first the furniture and then the action or the other way around. I like first Furniture then action.